docs/application-developers/backuppolicy.md

```diff
@@ -0,0 +1,77 @@
+# Backup Policy
+
+The `BackupPolicy` controller manages the scheduling and lifecycle of `Backup` resources in a Kubernetes cluster. It ensures backups are created according to a defined schedule and retains or deletes old backups based on a specified retention policy.
+
+## Features
+
+### Scheduling
+
+Create backups based on a configured schedule.
+
+### Retention Policy
+
+Allows defining retention policies based on either the number of backups to retain or the maximum age of backups. You can retain backups for a specified number or define a maximum age for backups (e.g., 7 days). Backups older than this period will be deleted. This age is calculated based on the creation timestamp of the backup. It supports multiple duration types: `hour`, `day`, `month`, and `year`.
+
+> More features will be added to the policy in future such as `Compression`, `Notifications`, `Encryption`
+
+## Configuration
+
+`BackupPolicy` consists of different configuration that manages the backup scheduling and other features.
+
+### ScheduleConfiguration
+
+The ScheduleConfiguration defines the schedule type to be used. We currently support Cron configuration, which is the most common method for creating schedules. More types may be supported in the future based on user needs.
+
+The configuration allows you to specify a timezone for calculating the schedule. The default is `UTC`.
+The config gives the ability to specify timezone to calculate the schedule based upon, the default is `UTC`.
+*Sample*
+
+```yaml
+scheduleConfig:
+    scheduleType: Cron
+    timezone: "CET"
+    Cron:
+        expression: "0 0 * * *"
+```
+
+### RetentionConfiguration
+
+The RetentionConfiguration defines the policy for retaining and deleting old backups. It ensures that backups are kept for a specified duration or up to a specified number of backups.
+
+#### Retention by Count
+
+With retention by count, you define the maximum number of backups to retain. Backups exceeding this limit are deleted, starting with the oldest ones (based on creation timestamp).
```

Comment:
Suggestion: Try formulating documentation in an active mood. I'm using this paragraph as an example, but my suggestion applies throughout.

> The controller(/a8s framework) will delete Backups above this limit, starting with the oldest ones.